Sources/SettingsFeature/Settings.swift

```diff
@@ -326,16 +326,15 @@ public let settingsReducer = Reducer<SettingsState, SettingsAction, SettingsEnvi
             )
           )
         )
-        .debounce(id: CancelId(), for: 1, scheduler: environment.mainQueue)
         .fireAndForget(),
         environment.apiClient.refreshCurrentPlayer()
           .catchToEffect()
           .map(SettingsAction.currentPlayerRefreshed)
       )
+      .debounce(id: UpdateRemoteSettingsId(), for: 1, scheduler: environment.mainQueue)
```

Comment:
We wanna debounce the entire concatenated of the effect to avoid loading the remote settings overly often.